### PR TITLE
Adds debian bullseye packaging image among other supported operating system images

### PR DESCRIPTION
**Pull request by gurkanindibay**
Installed with using packagecloud installation scripts 
Need to change deb.sh for community nightlies and enterprise
image

### CODE REVIEW

File: os-list.csv [hunk 1]
```diff
@@ -2,6 +2,7 @@ centos,8
 centos,7
 debian,buster
 debian,stretch
+debian,bullseye
```
Comment (gurkanindibay) [inline os-list.csv:5]: After adding this line I only executed ./update_dockerfiles

Comment (onurctirtir) [reply os-list.csv:5]: great

**Review by onurctirtir — APPROVED** [review]
> Need to change deb.sh for community nightlies and enterprise

True. Plus, I'm wondering should we make some changes in `citus_package` to start building packages for this distro, maybe a small matrix change or something like that .

**Comment by gurkanindibay** [comment]
> > Need to change deb.sh for community nightlies and enterprise
> 
> True. Plus, I'm wondering should we make some changes in `citus_package` to start building packages for this distro, maybe a small matrix change or something like that .

Yes I will send other PR's to add this support for pipelines

**Pull request merged**